library/src/hiprand/hiprand_clang.cpp

```diff
@@ -0,0 +1,382 @@
+// Copyright (c) 2017 Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
What is this file for?
I always bug people about copyright dates.

Reply:
Good question. So, there's a part of the code where a file with hiprand_${HIP_PLATFORM} is used to read a bunch of hiprand functions that differ from hip and nvcc. In the library/src/hiprand directory, there are currently 2 files- hiprand_hcc.cpp and hiprand_nvcc.cpp. 

I just copied and pasted the hcc file and renamed it to _clang.cpp, so we can have both covered. I know, not an elegant solution, but it's the only one I figured out. I will fix the copyright dates.

Reply:
Just curious, is there anything HCC-specific in hiprand_hcc.cpp/hiprand_hcc.hpp ? In #116 I just made cmake point to hiprand_hcc.* when `${HIP_PLATFORM}` is `clang`

Reply:
No there is isn't, so your comment is noted that we shouldn't have two files.

Reply:
@amdkila don't bother with copyright dates, we should be able to have one file.